batch/batch/worker/worker.py

```diff
@@ -167,6 +167,8 @@ def compose(auth: Union[MutableMapping, str, bytes], registry_addr: str = None):
 
 image_lock = aiorwlock.RWLock()
 
+DISK_OPERATION_TIMEOUT_SECONDS = 10 * 60
```

Comment:
do these operations actually take ten minutes? That seems... wrong? Do these endpoints not return immediately? Do they wait for the disk to actually be created or deleted?

Reply:
In GCP, it can take a really long time. I'm not sure why. We could start with 5 minutes?